packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.tsx

```diff
@@ -23,19 +23,18 @@ const AccordionSectionHeader = ({
   <div
     className={cx(
       "p-600 bg-white-normal flex items-center",
-      expanded ? "min-h-[19px]" : "min-h-[44px]",
+      expanded ? "min-h-[19px]" : "min-h-form-box-normal",
```

Comment:
It's just replacing a custom value with an already existing token (and semantically it makes sense). Right?

Reply:
Although the linter suggests so, we want to refrain from using component specific tokens outside of their context. In this case, a token for Forms should not be used in Accordion